### DISCUSSION TITLE
Qual destes cursos é a melhor para iniciar?

### DISCUSSION THREAD

**Discussion by babybeibe** [frontendbr/forum#2228] [Perguntas e Respostas]
Boa tarde!

Atualmente trabalho como Designer Gráfico, mas busco me tornar um desenvolvedor Front-end. :) 
Encontrei duas opções de curso que aparentemente são bem atrativas e gostaria de uma opinião sobre qual seria a melhor escolha.

Estes são os cursos:
- E.B.A.C. | CURSO ONLINE DE Profissão: Engenheiro Front-end
https://ebaconline.com.br/front-end-profession

- Origamid | Todos os cursos do plano vitalício
https://www.origamid.com/

Gostaria muito saber qual seria a melhor opção para quem esta querendo iniciar nesta carreira e ter o melhor aprendizado de acordo com as necessidades do mercado.

Caso tenham alguma outra sugestão de instituição/curso desta mesma modalidade agradeço também!

Obrigado desde já!

**Comment by gabriel-hahn** [comment]
Olá!! Esse seria seu primeiro curso em desenvolvimento ou você já tem uma certa experiência? Uma dica que acho legal é fazer um curso bem "genérico" de primeira, assim você aprende conceitos básicos de web e também entende como funciona as linguagens, como o Front end se comunica com uma API e também para você realmente ver se o Front end é algo que você realmente gostaria de fazer e se aprofundar nos estudos antes mesmo de investir pesado em um curso mais caro.

Não conheço esses cursos que você mencionou e não saberia opinar sobre eles, mas existe uma plataforma que acho muito legal que é a Udemy, no qual os cursos estão sempre em promoção entre 25-35 reais (não compre no valor cheio, sempre tem promoção) e possuem cursos de ótima qualidade. Algumas dicas seriam este [curso](https://www.udemy.com/course/the-complete-web-development-bootcamp/) caso o inglês não seja um problema para você ou esse [aqui](https://www.udemy.com/course/curso-web/). Esses cursos vão te dar uma boa abrangência nos assuntos e esse tipo de curso me deu uma boa base no início da minha carreira como desenvolvedor. São cursos beeeem longos, mas valem a pena para depois se aprofundar em uma linguagem e framework específico etc. No Youtube também existe conteúdos ótimos, mas ainda assim prefiro a Udemy por ser valores acessíveis e o conteúdo é um pouco mais "filtrado", já que a sopa de letrinhas da área pode ser meio confusa no início haha ☺️

Existe também o Discover da Rocketseat que é um conteúdo gratuito focado em assuntos da base da programação web e é bem interessante, recomendo.

Depois de fazer um curso desses, recomendaria tentar fazer 1 ou 2 projetos pessoais, pode ser qualquer coisa, mas isso irá te ajudar muito em entender como solucionar problemas quando não existir um curso te guiando, além de criar um portfólio para tua primeira oportunidade o/

**Comment by theandersonn** [comment]
Para começar recomendo `Origamid `.
Já que você tem uma base boa de Design Gráfico irá se identificar com os cursos voltados para UX/UI, Tipografia, adquirindo base para montar as interfaces e o código.

Super recomendo.
